pinot-broker/pom.xml

```diff
@@ -153,5 +153,11 @@
       <artifactId>mockito-core</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
```

Comment:
Do we need this dependency? I feel we only need to mock `millis()`

Reply:
It is easier to advance clock and stuff. I can remove it if we are concerned about dependency